flake.nix

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
are this files needed?

Reply:
can we remove them?

Reply:
we absolutely can, but for anyone using nixos, nix-darwin, or just home-manager, it's incredibly useful to have. For anyone who isn't using those, it's harmless. Your call.

Reply:
Why not add flake.nix to .gitignore, so you don't have to think about removing it in future?

Reply:
yup. I know how to use gitignore. I thought it might be useful for others who, like me, use nix*. It's been removed though, so can we merge this now?